### DISCUSSION TITLE
Alex Arena - Interval.com

### DISCUSSION THREAD

**Discussion by just-be-dev** [devtools-fm/devtools.fm#127] [Announcements]
## Background

- https://interval.com/
- [Building a bookmarks CMS with internval](https://www.youtube.com/watch?v=K4pddUlQ9CQ)
- https://twitter.com/alexarena

## Themes

- No Code vs Only Code
- Internal apps

## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey everyone! Our guest today is Alex Arena who's the founder of interval.com. Interval is described as a batteries-included approach to building rich internal tools directly in your app's backend codebase. Sounds really interesting and I'm excited to dig in. Before we do though, Alex, would you like to tell our listeners a bit more about yourself?

## Questions

- [x] Give us the elevator pitch. What's interval and why would someone want to use it?
- [x] There's an increasing amount of products targeted at building internal tools on the market. What need do you think interval fills that the others don't?
- [x] Interval is a low-code tool instead of a no-code tool. What inspired this approach?
- [x] Interval code runs in a customer's backend but the input is provided from your frontend. How to you make sure that's secure?
- [x] What's been the hardest challenge of building interval?
- [x] How much dog fooding do you do inside interval? Are your own internal tools built with it?
- [x] Are you thinking about potentially allowing users to embed interval forms directly inside their products?

## Future
- [x] What's next for interval?
- [x] What do you think the future of dev-tooling looks like?

**Comment by just-be-dev** [comment]
Tooltips:
- https://clay.earth/
- https://hypertalk.substack.com/p/hello-hypertalk

**Comment by alexarena** [comment]
Tooltips:
- [Code spell checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
- [Ultimate ffmpeg guide](https://img.ly/blog/ultimate-guide-to-ffmpeg/#introduction-to-ffmpeg)

**Comment by hipstersmoothie** [comment]
https://dotenv.org/vault
https://tauri.app/
